Import-VIRole.ps1

```diff
@@ -118,9 +120,16 @@ function Import-VIRole
                 Write-Warning -Message "Permission named $(($_.Exception.Message.Split("'"))[1]) not found"
             }
         }
-
+	if (!$Overwrite)
+	{
```

Comment:
Fix indentation here and below. The rest of the file uses a 4 space indent per level.